docs/design/noc-root-cert-design.md

```diff
@@ -15,48 +18,64 @@ To distinguesh NOC root certificates from others, an `isNOC` boolean field will
 ## Transactions
 
 ### 1. ADD_NOC_X509_ROOT_CERTIFICATE
-This transaction adds a NOC Root Certificate owned by the Vendor.
+This transaction adds a NOC root certificate owned by the Vendor.
 
 - Who can send: Vendor account
-  - VID-scoped NOC Root Certificate: The `vid` field in the certificate's subject must be equal to the Vendor account's VID.
 - Validation:
   - The provided certificate must be a root certificate:
```

Comment:
Please mention, that the certificate must have a VID field set (https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/proto/pki/certificate.proto#L26), and must be equal to the Vendor Account's VID.

Reply:
As I understand, the provided PEM certificate does not include a VID field. Please correct me if I am mistaken

Reply:
Yes, but we have an optional VID field in certificate.proto (for VID scoped certs we require that a VID in PEM is equal to the VID field in the certificate).
So, the question is should we require a VID field to be a must-have for NOC root transaction. I think yes, and if so, the provided VID value must be equal to the Account's VID.

Reply:
Corrected the transaction description and added a question regarding the necessity of including the VID parameter in this transaction.